### DISCUSSION TITLE
Text clipping on long settings section headers

### DISCUSSION THREAD

**Discussion by dunsparce9** [ppy/osu#26288] [Q&A] [answered]
![2023-12-31 16_53_17-osu!](https://github.com/ppy/osu/assets/66077935/5af54d9a-6623-4efa-a5c3-712fcd552a22)

I don't know if the solution would be to have the text slowly scroll or to just make the entire header a bit wider.

**Comment by Joehuu** [comment, marked as answer]
Already tracked in https://github.com/ppy/osu/issues/21617.
